templates/settings.html

```diff
@@ -1,22 +1,27 @@
 <!DOCTYPE html>
 <html>
+
 <head>
```

Comment:
as said before, it would be very helpful to straight up just have a boilerplate/layout template as it seems fairly repetitive